src/player.ts

```diff
@@ -235,6 +235,19 @@ export class Player extends NativeInstance<PlayerConfig> {
     PlayerModule.seek(this.nativeId, time);
   };
 
+  /**
+   * Shifts the time to the given `offset` in seconds from the live edge. The resulting offset has to be within the
+   * timeShift window as specified by `maxTimeShift` (which is a negative value) and 0. When the provided `offset` is
+   * positive, it will be interpreted as a UNIX timestamp in seconds and converted to fit into the timeShift window.
+   * When the provided `offset` is negative, but lower than `maxTimeShift`, then it will be clamped to `maxTimeShift`.
+   * Has no effect for VoD.
+   *
+   * Has no effect if there is no active playback session.
```

Comment:
There is no playback session concept on iOS so I would adapt this a bit.

```suggestion
   * Has no effect if the player is unloaded.
```

Reply:
What about: 

```suggestion
   * Has no effect if no sources are loaded.
```

Reply:
I feel like it should be understandable. I'm going with the "Has no effect if no sources are loaded" version.